notebook/Solution/04 - LSTM -Multivariate Input -Univariate Output-MultiStep.ipynb

```diff
@@ -0,0 +1,2024 @@
+    "y_test_prediction_demand = scaler.inverse_transform(y_test_prediction_demand.detach().numpy())\n",
+    "y_test_demand = scaler.inverse_transform(testY_demand.detach().numpy())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 70,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "y-test_demand\t\t\t\ty-predict_demand\n",
+      "[234.50386 234.72456]\t\t[268.7812  369.76947]\n",
+      "[234.72456 230.06245]\t\t[267.70892 311.34174]\n",
+      "[230.06245 357.39447]\t\t[290.16986 244.27534]\n",
+      "[357.39447 259.24603]\t\t[340.61908 235.7148 ]\n",
+      "[259.24603 244.23433]\t\t[301.31244 211.71284]\n",
+      "[244.23433 402.60736]\t\t[301.29877 224.51753]\n",
+      "[402.60736 255.0605 ]\t\t[289.0722 236.8769]\n",
+      "[255.0605 342.6054]\t\t[306.59564 235.03902]\n",
+      "[342.6054 268.6386]\t\t[367.61517 233.67964]\n",
+      "[268.6386  188.60152]\t\t[322.74408 212.83784]\n",
+      "[188.60152 202.02339]\t\t[266.03705 218.2148 ]\n",
+      "[202.02339 213.50777]\t\t[269.59174 272.61127]\n",
+      "[213.50777 316.8476 ]\t\t[304.61713 246.78316]\n",
+      "[316.8476  286.41205]\t\t[368.30658 235.53511]\n",
+      "[286.41205 303.4472 ]\t\t[348.8847  218.73628]\n",
+      "[303.4472  304.94916]\t\t[372.32025 235.5644 ]\n",
+      "[304.94916 331.90033]\t\t[377.373   263.01556]\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"y-test_demand\\t\\t\\t\\ty-predict_demand\")\n",
+    "for i in range(len(y_test_prediction_demand)):\n",
+    "    print(f\"{y_test_demand[i]}\\t\\t{y_test_prediction_demand[i]}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 71,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "y_test_demand_shape : (17, 2)\n",
+      "y_test_demand_pred_shape : (17, 2)\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(f\"y_test_demand_shape : {y_test_demand.shape}\")\n",
+    "print(f\"y_test_demand_pred_shape : {y_test_prediction_demand.shape}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 72,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Train Score: 52.14 RMSE\n",
+      "Test Score: 62.16 RMSE\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Calculate root mean squared error\n",
+    "trainScore_demand = math.sqrt(mean_squared_error(y_train_demand[:,0], y_train_prediction_demand[:,0]))\n",
+    "print('Train Score: %.2f RMSE' % (trainScore_demand))\n",
+    "testScore_demand = math.sqrt(mean_squared_error(y_test_demand[:,0], y_test_prediction_demand[:,0]))\n",
+    "print('Test Score: %.2f RMSE' % (testScore_demand))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Plot Explanation\n",
+    "- Blue Line-Indicate as test data before sequencing \n",
```

Comment:
Again, believe you can come up with a better explanation. Thanks. Now it is not really intuitive.